pontoon/base/models.py

```diff
@@ -108,11 +107,17 @@ def user_profile_url(self):
 
 def user_gravatar_url(self, size):
     email = hashlib.md5(self.email.lower().encode("utf-8")).hexdigest()
-    data = {"s": str(size)}
-
-    if not settings.DEBUG:
-        append = "_big" if size > 88 else ""
-        data["d"] = settings.SITE_URL + static("img/anon" + append + ".jpg")
+    data = {
+        "s": str(size),
+        "d": "https://ui-avatars.com/api/{name}/{size}/{background}/{color}".format(
+            name=quote(
+                self.display_name if self.display_name is not "" else "Imported"
```

Comment:
This is the corner case where I logged in with pontoon generated AnonymousUser and it doesnt have first_name, email etc. So just to handle this add if for display_name.

Reply:
Thanks for checking so deep! AnonymousUser is not supposed to be used as the actual use logging into Pontoon, so we can remove this. Also, if we were to keep this, I think you could simplify the code to: `self.display_name or "Imported"`.

Reply:
Is every user is goint to have first_name or email, then we are good. Removed the fallback.